### DISCUSSION TITLE
Combined Polar / Smith Diagram

### DISCUSSION THREAD

**Discussion by iwbnwif** [ra3xdh/qucs_s#499] [Q&A] [answered]
I am raising this as a discussion rather than an issue because it might be down to my lack of knowledge!

The combined Polar / Smith in Qucs-S seems to be inverted when compared to the original Qucs and also QucsStudio. 

More specifically, the grid lines for the Smith portion are on the top and the Polar grid lines are in the lower half. However, the graph plots appear to be in the right place, so the Smith graph overlays the Polar grid and vice-versa. Furthermore, for some cases, the polar graph appears to be plotted over both the Polar and Smith grids.

Is this the expected behavior?

![image](https://github.com/ra3xdh/qucs_s/assets/11036083/a4da9e0c-8c9f-4eb0-8cb4-4eefd67f4408)

**Comment by ra3xdh** [comment, marked as answer]
I have just tested unofficial Qucs-0.0.20 release candidate and found the same appearance for Smith/Polar chart. The most probably the changes in these charts were made after 0.0.19 release. I cannot say what was its purpose. The new appearance of Smith/Polar is surely not a bug introduced by Qt5 porting. QucsStudio was forked from Qucs around 2010 and doesn't share the latest changes in code.

**Reply by iwbnwif** [reply]
Thank you. It is probably my misunderstanding of the intended function - I will keep reading the docs!
